### PR TITLE
link-out icon improved

### PR DESCRIPTION
**Pull request by tinycosmos**
Also added a button which links to tutorial page
Screenshot (90)

### CODE REVIEW

**Comment by tinycosmos** [comment]
@yochannah please have a look
or shall I close this PR

File: static/style.less [hunk 1]
```diff
@@ -21,6 +21,10 @@
   transition-delay: 0.5s;
   transition-duration: 0.6s;
 }
+.tutorialsec{
```
[the remark this answers is not in the document]
Comment (tinycosmos) [reply static/style.less:24]: oh okay maybe i just forgot to undo some changes that i tried to do for previous issues.

File: static/style.less [hunk 2]
```diff
@@ -3,7 +3,7 @@
 //Yo likes prepros: https://prepros.io/.
 @intermine-color: #bed73b;
 @seriously-pale-grey: #e9e9e9;
-@spacer: 1em;
+@spacer: auto;
```
[the remark this answers is not in the document]
Comment (tinycosmos) [reply static/style.less:6]: Uhm okay cool

File: static/style.css [hunk 3]
```diff
@@ -1,1318 +1,1536 @@
 .transition {
-  transition-property: all;
-  transition-delay: 0.5s;
-  transition-duration: 0.6s;
+    transition-property: all;
+    transition-delay: 0.5s;
+    transition-duration: 0.6s;
 }
+
```
[the remark this answers is not in the document]
Comment (tinycosmos) [reply static/style.css:6]: As i previously mentioned once, these changes are caused by the editor automatically i guess :sweat_smile:

File: layouts/partials/linksbiologists.html [hunk 4]
```diff
@@ -15,3 +15,9 @@ <h4 class="categories biologists">Biologists</h4>
                 <use xlink:href="{{.Site.BaseURL}}/images/symbol-defs.svg#icon-externallink"></use></svg>
             </a></li>
 </ul>
+<script>
```
[the remark this answers is not in the document]
Comment (tinycosmos) [reply layouts/partials/linksbiologists.html:18]: yeah i thought of something like this, but couldn't find out the way
Cool, i can try this out.

**Comment by uosl** [comment]
Like Yo says, this should probably be done some other way (like partials or maybe **even better**, [shortcodes](https://gohugo.io/content-management/shortcodes/) as it might not be obvious to people working on the code that there's a JS snippet that adds link-out icons). Closing as it's been a while; appreciate the PR nonetheless, Roshni. :smiley:

**Pull request closed without merging**